[2025-07-16T14:33:55.748+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-07-16T14:33:55.778+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_verify.task_3 scheduled__2025-07-15T00:00:00+00:00 [queued]>
[2025-07-16T14:33:55.791+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_verify.task_3 scheduled__2025-07-15T00:00:00+00:00 [queued]>
[2025-07-16T14:33:55.821+0000] {taskinstance.py:2866} INFO - Starting attempt 3 of 4
[2025-07-16T14:33:55.845+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): task_3> on 2025-07-15 00:00:00+00:00
[2025-07-16T14:33:55.857+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_verify', 'task_3', 'scheduled__2025-07-15T00:00:00+00:00', '--job-id', '63', '--raw', '--subdir', 'DAGS_FOLDER/dag_verify.py', '--cfg-path', '/tmp/tmpabofy13y']
[2025-07-16T14:33:55.863+0000] {standard_task_runner.py:105} INFO - Job 63: Subtask task_3
[2025-07-16T14:33:55.866+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=293) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-16T14:33:55.868+0000] {standard_task_runner.py:72} INFO - Started process 294 to run task
[2025-07-16T14:33:55.948+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_verify.task_3 scheduled__2025-07-15T00:00:00+00:00 [running]> on host 4461bcf72552
[2025-07-16T14:33:56.102+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_verify' AIRFLOW_CTX_TASK_ID='task_3' AIRFLOW_CTX_EXECUTION_DATE='2025-07-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-07-15T00:00:00+00:00'
[2025-07-16T14:33:56.103+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-07-16T14:33:56.503+0000] {logging_mixin.py:190} INFO - Matplotlib version: 3.9.2
[2025-07-16T14:33:56.504+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-07-16T14:33:56.527+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-07-16T14:33:56.528+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_verify, task_id=task_3, run_id=scheduled__2025-07-15T00:00:00+00:00, execution_date=20250715T000000, start_date=20250716T143355, end_date=20250716T143356
[2025-07-16T14:33:56.606+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-07-16T14:33:56.632+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-16T14:33:56.666+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
